### PR TITLE
Add Plotly transform

### PR DESCRIPTION
**Pull request by theengineear**
Adding a Plotly transform to handle the `application/vnd.plotly.v1+json` mimetype.

Closes: #165

### CODE REVIEW

**Comment by theengineear** [comment]
^^ @rgbkrk rebased and flow is passing locally

**Comment by rgbkrk** [comment]
Rockin' thanks!

File: packages/frontend/components/contents/index.js [hunk 1]
```diff
@@ -21,9 +21,13 @@ import type { Content } from "./types";
 
 // $FlowFixMe: our flow config isn't picking up modules that package separate .js.flow files
 import DataResourceTransform from "@nteract/transform-dataresource";
+import { PlotlyNullTransform, PlotlyTransform } from "../../transforms";
 
+// Order is important here. The last transform in the array will have order `0`.
 const { transforms, displayOrder } = [
-  DataResourceTransform
+  DataResourceTransform,
+  PlotlyNullTransform,
```
Comment (rgbkrk) [inline packages/frontend/components/contents/index.js]: I love this.

File: packages/frontend/transforms/PlotlyTransform.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+const NULL_MIMETYPE = "text/vnd.plotly.v1+html";
+const MIMETYPE = "application/vnd.plotly.v1+json";
+
+/*
+ * As part of the init notebook mode, Plotly sneaks a <script> tag in to load
+ * the plotlyjs lib. We have already loaded this though, so we "handle" the
+ * transform by doing nothing and returning null.
+ */
```
Comment (rgbkrk) [inline packages/frontend/transforms/PlotlyTransform.js]: ❤️

File: packages/frontend/components/contents/index.js (hunk 1 above)
Comment (theengineear) [inline packages/frontend/components/contents/index.js]: Ha, yah, couldn't think of a better name for it. PlotlyCheatyHideThingsTransform?

**Comment by rgbkrk** [comment]
Ok, nowwwww we'll ship with plotly support.

File: packages/frontend/pages/_document.js [hunk 3]
```diff
@@ -64,6 +64,8 @@ class MyDocument extends Document {
             type="text/css"
           />
           <link rel="stylesheet" type="text/css" href="/static/commuter.css" />
+
+          <script src="https://cdn.plot.ly/plotly-latest.min.js" />
```
Comment (rgbkrk) [inline packages/frontend/pages/_document.js]: We may want to bundle this later so that people can run offline.

**Pull request merged**